com.basistech.m2e.code.quality.pmd/src/main/java/com/basistech/m2e/code/quality/pmd/EclipsePmdProjectConfigurator.java

```diff
@@ -107,12 +108,20 @@ protected void handleProjectConfigurationChange(
 		final MavenPluginConfigurationTranslator pluginCfgTranslator =
 		        MavenPluginConfigurationTranslator.newInstance(maven,
 		                session, mavenProjectFacade.getMavenProject(monitor),
-		                pmdGoalExecution, project, monitor);
-
+		                execution, pmdGoalExecution, project, monitor);
 		this.createOrUpdateEclipsePmdConfiguration(mavenPluginWrapper, project,
 		        pluginCfgTranslator, monitor, session);
-
+	
```

Comment:
careful you introduced some whitspace on an empty line :)

Reply:
@victornoel if you're into this sort of thing, fancy contributing a Checkstyle failing the build for WS ?

Reply:
@vorburger should we start with a minimal setup with forbidding such thing or do you know of a checkstyle configuration that we could use as a starter?

Reply:
let's do this in #156